### DISCUSSION TITLE
Having access to the quarto command in command line

### DISCUSSION THREAD

**Discussion by ark561** [quarto-dev/quarto-cli#5166] [Q&A] [answered]
Hello,

I am a bit inexperienced working with GitHub, but am trying to learn more.

I've got a bit of confusion on how to actually publish my documents using GitHub Pages. I've been following the guide at https://quarto.org/docs/publishing/github-pages.html, but when I get to the "Render to docs" section, I get an error when I type 

`quarto render`

I'm using a terminal on Mac, and get the following error:
`zsh: command not found: quarto`

How to I gain access to the function for quarto?

I know this is a very simple question, but how do I do this? Thank you in advance for your patience and help!

**Comment by mcanouil** [comment]
I am assuming you are using RStudio as your IDE, am I correct?
(RStudio embed a version of Quarto)

If that is the case, I don't think RStudio is adding its internal Pandoc or Quarto to the PATH which is required if you want to use it in the command line.
I saw something about this, but don't recall where.
This is possibly something to report to <https://github.com/rstudio/rstudio>.

Meanwhile, what you can do is installing Quarto yourself by downloading the proper version for your OS at <https://quarto.org/docs/download/>.

**Reply by cderv** [reply, marked as answer]
Just to add to that answer: 

When using RStudio IDE bundled version, and when terminal is needed, best things is to use the RStudio IDE built-in terminal where `quarto` command should be available. At least with a recent verison of the RStudio IDE.

**Reply by mcanouil** [reply]
Oh, so it has been fixed (at least internally).

**Reply by ark561** [reply]
Oh wow I didn't even think about that - I feel like such a tool!

Thank you so much for your help!
